rpc-jobs/RPC-Artifact-Build.yml

```diff
@@ -92,7 +96,15 @@
           auth-id: "github_account_rpc_jenkins_svc"
           status-context: 'CIT/artifact-{image}-{context}'
     dsl: |
-      // CIT Slave node
+      node('ArtifactBuilder2') {{
+        dir("rpc-gating") {{
+            git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
+            common = load 'pipeline-steps/common.groovy'
+            aio_prepare = load 'pipeline-steps/aio_prepare.groovy'
```

Comment:
Could you import aio_prepare in artifact_build.groovy since that's where it's used?  And is there even a need to run aio_prepare.prepare() for this apt piece?